src/apps/intel/intel_app.lua

```diff
@@ -116,6 +148,11 @@ function Intel82599:push ()
       end
    end
    self.dev:sync_transmit()
+   if self.dev.txstats then
```

Comment:
How frequently do these counters need to be synchronized with hardware?

This code is synchronizing them every breath but that adds up to a significant number of PCIe accesses even e.g. for NICs that are completely idle. This may cause performance degradation in scenarios that we don't have CI performance coverage on at the moment e.g. app network with very many NICs where most are idle but some are active.

One alternative would be to use a timer to update every <interval> e.g. one millisecond.